project1/templates/register.html

```diff
@@ -10,9 +10,15 @@
 <body>
     <form action="{{url_for('display')}}" class="border border-light p-5" method="POST">
         <p class="h4 mb-4 text-center">Register</p>
+
         <input name="Email" type="email" id="defaultLoginFormEmail" class="form-control mb-4" placeholder="E-mail">
         <input name="password" type="password" id="defaultLoginFormPassword" class="form-control mb-4"
             placeholder="Password">
-        <button class="btn btn-info btn-block my-4" type="submit">Register Me</button>
+        <button class="btn btn-info btn-block my-4" type="submit">Sign Up</button>
+        <div class="text-center">
+            <p>Already a member?
+                <a href="{{url_for('login')}}">Sign In</a>
+            </p>
+        </div>
```

Comment:
Try implementing using button instead of anchor tag, check for resources on "formaction" for a button in form.

Reply:
Updated Sir.